src/ChainSafe.Gaming.Reown/Models/WalletModel.cs

```diff
@@ -0,0 +1,30 @@
+using Newtonsoft.Json;
+
+namespace ChainSafe.Gaming.Reown.Models
+{
+    /// <summary>
+    /// Reown wallet model used for identifying and redirecting wallets.
+    /// </summary>
+    public class WalletModel
+    {
+        [JsonProperty("id")] public string Id { get; set; }
+
+        [JsonProperty("name")] public string Name { get; set; }
+
+        [JsonProperty("homepage")] public string Homepage { get; set; }
+
+        [JsonProperty("image_id")] public string ImageId { get; set; }
```

Comment:
is this like image link?

Reply:
No, they use their new dedicated API now for all wallet-related data distribution. Below is the endpoint we should be sending requests to. It's improtant to use `ReownHttpClient` for this as it sets the required HTTP headers.

https://api.web3modal.com/getWalletImage/{wallet.ImageId}